regtech_mail_api/api.py

```diff
@@ -77,10 +77,10 @@ async def get_debug_info(request: Request):
 async def send_email(request: Request):
     headers = request.headers
     subject = headers["X-Mail-Subject"]
-    sender_addr = headers["X-Mail-Sender-Address"]
-    sender_name = headers["X-Mail-Sender-Name"]
+    sender_addr = request.user.email
+    sender_name = request.user.name
```

Comment:
Is this guaranteed to be set even when the users hasn't completed their user profile? I'm guessing yes, since the test seems to test that case, but just wanted to confirm there's not risk of raising an exception here.

Reply:
It’ll either be the name or an empty string  if that name isn’t in the token